calblueprint/ogsc
# Add display components for player profile Attendance tab

**⚠️ Depends on #38.** 

* Adds absence table components
* Assorted Tailwind updates from hi-fi mockups
  * Introduces new colors and muted companions
  * Adds Montserrat to replace Airbnb Cereal
  * Adds `navigation-tab` / `navigation-tab-highlighted` standard class for buttons
  * Updates sidebar theme and includes logo
* Moves player/[id].tsx page to `[role]/player/[id].tsx` page. This will allow this page to become generic for all different roles that can view a player profile.

## How to Test/Use PR feature

* Follow Test Plan for #38 to migrate and seed your local database.
* Login as an admin user and navigate to the Players page (first sidebar option).
* Click on any one of the players.
* Navigate to the Attendance tab.
* Verify that absences appear here, correctly categorized and sorted.

## PR Checklist

Does your branch (check if true):

## Related PRs

Depends on #38.

## Screenshots

![image](https://user-images.githubusercontent.com/2977329/98501354-c927b100-2203-11eb-9c96-fd1701a1e000.png)


CC: @erinysong
